cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -353,123 +377,121 @@ func (s *Server) newRelease(ctx context.Context, packageRef *corev1.AvailablePac
 	var values map[string]interface{}
 	if valuesString != "" {
 		values = make(map[string]interface{})
-		err = yaml.Unmarshal([]byte(valuesString), &values)
-		if err != nil {
+		if err = json.Unmarshal([]byte(valuesString), &values); err != nil {
```

Comment:
Just curious, why did you replace the `yaml` dep with the `json` one here (and in `updateRelease(...)`)

Reply:
Don't remember *exact* reason. It's been a few days. This business with "Values" took especially long due to how Flux expects it (some kind of v1.JSON) and how we handle it. So somewhere in the  (very long debugging process) I arrived at this. Possibly, I saw when I was working on this "values" business that it was JSON I was unmarshaling not YAML and to use encoding/json was clearer and it also happened to work :-)